.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-  "printWidth": 80,
+  "printWidth": 120,
```

Comment:
Less wrapping, especially since https://github.com/prettier/prettier/issues/3368

Reply:
I'm a little torn. When it comes to template strings, whitespace can/does matter sometimes so it would seem best for prettier to not auto format.

However, I've tried 120 on a couple projects in the past and always revert to 80. The reason being is that prettier will take "short lines" that I thought I formatted in a nice way and then move them up to the same line. (Kent C Dodds discusses [here](https://youtu.be/hPI9UnE4dws?t=459) briefly).

Mind if we leave at 80? If we do switch to 120 then I would wan't to preserve the git history - https://medium.com/millennial-falcon-technology/reformatting-your-code-base-using-prettier-or-eslint-without-destroying-git-history-35052f3d853e

Reply:
We can leave this one to the back burner for now. I think I have an idea of what you may have experienced but this is no biggie. I can go either way. Maybe I'll have a better opinion after I watch the video you linked.